package.json

```diff
@@ -13,9 +13,19 @@
         "Swagger"
     ],
     "private": false,
+    "types": "dist/index.d.ts",
     "files": [
-        "dist"
+        "dist",
+        "*.d.ts",
+        "basic",
+        "openapi"
     ],
+    "exports": {
+        ".": "./dist/index.js",
+        "./basic": "./dist/flavors/basic.js",
+        "./openapi": "./dist/flavors/openapi.js",
```

Comment:
Every sub-package export's root source must be listed here. This is the same value that is defined in the `main` field of the sub-directory `package.json` files. It's possible that the value in the sub-`package.json` can be omitted; the value here cannot be.